addons/project/views/project_project_views.xml

```diff
@@ -558,13 +558,33 @@
             </field>
         </record>
 
+        <record id="project_project_view_form_in_calendar" model="ir.ui.view">
+            <field name="name">project.project.view.form.calendar</field>
+            <field name="model">project.project</field>
+            <field name="inherit_id" ref="edit_project"/>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <xpath expr="//sheet" position="after">
+                    <footer>
+                        <button string="Save &amp; Close" special="save" data-hotkey="q" class="btn btn-primary" close="1"/>
+                        <button string="Discard" special="cancel" data-hotkey="x" class="btn btn-secondary" close="1"/>
+                        <button name="action_view_project" type="object" title="View task"
```

Comment:
I missed it on first review, but shouldn't this button title be 'view project' instead ?